package.json

```diff
@@ -64,49 +64,49 @@
     "test:linux-browser-compatibility-other": "cd ./integration/js && npm run test -- browser_compatibility linux_other "
   },
   "devDependencies": {
-    "@fluffy-spoon/substitute": "^1.89.0",
-    "@types/chai": "^4.1.7",
-    "@types/chai-as-promised": "^7.1.0",
+    "@fluffy-spoon/substitute": "^1.208.0",
+    "@types/chai": "^4.3.1",
+    "@types/chai-as-promised": "^7.1.5",
     "@types/mocha": "^5.2.6",
-    "@types/sinon": "^7.0.12",
-    "@types/uuid": "^8.3.1",
-    "@typescript-eslint/eslint-plugin": "^4.18.0",
-    "@typescript-eslint/parser": "^4.18.0",
-    "chai": "^4.2.0",
+    "@types/sinon": "^7.5.2",
+    "@types/uuid": "^8.3.4",
+    "@typescript-eslint/eslint-plugin": "^4.33.0",
+    "@typescript-eslint/parser": "^4.33.0",
+    "chai": "^4.3.6",
     "chai-as-promised": "^7.1.1",
     "chai-fetch-mock": "^3.0.0",
     "cross-env": "^6.0.3",
-    "eslint": "^7.22.0",
+    "eslint": "^7.32.0",
     "eslint-config-prettier": "^6.15.0",
-    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-prettier": "^3.4.1",
     "eslint-plugin-simple-import-sort": "^5.0.3",
     "esm": "^3.2.25",
-    "fetch-mock": "^9.10.7",
+    "fetch-mock": "^9.11.0",
     "fs-extra": "^8.1.0",
-    "git-rev-sync": "^3.0.1",
+    "git-rev-sync": "^3.0.2",
     "longjohn": "^0.2.12",
-    "mocha": "^9.1.2",
+    "mocha": "^9.2.2",
     "node-fetch": "^2.6.1",
     "nyc": "^15.1.0",
-    "prettier": "^2.1.2",
+    "prettier": "^2.6.2",
     "rimraf": "^3.0.2",
     "sinon": "^7.3.2",
     "spawn-wrap": "^2.0.0",
-    "ts-loader": "^8.0.18",
+    "ts-loader": "^8.4.0",
     "ts-node": "^9.1.1",
     "typedoc": "0.19.2",
-    "typescript": "^4.2.3"
+    "typescript": "^4.6.4"
   },
   "dependencies": {
```

Comment:
Thanks for updating dependency versions, but can we keep the original versions to scope this PR to the regionalized endpoint changes?

Reply:
yep, not sure how that happened